feature/gribi/ate_tests/base_hierarchical_route_installation_test/README.md

```diff
@@ -6,35 +6,48 @@ Validate IPv4 AFT support in gRIBI with recursion.
 
 ## Procedure
 
-*   Connect ATE port-1 to DUT port-1 and ATE port-2 to DUT port-2.
-*   Establish gRIBI client connection with DUT using default parameters.
-*   Using gRIBI Modify RPC install the following IPv4Entry set:
-    *   198.51.100.0/24 to NextHopGroup containing one NextHop, specified to be
-        203.0.113.1/32
-    *   203.0.113.1/32 to NextHopGroup containing one NextHop specified to be
-        the address of ATE port-2.
-*   Forward packets between ATE port-1 and ATE port-2 (destined to
+**Topology**
+
+*  Connect ATE port-1 to DUT port-1 and ATE port-2 to DUT port-2.
+    *   TODO: create a non-default VRF (VRF-1) that includes DUT port-1.
+
+**Validate hierarchical resolution**
+
+1.  Establish gRIBI client connection with DUT.
+1.  TODO: Use gRIBI Modify RPC to install entries per the following order, and ensure FIB ACK is received for each of the AFTOperation:
+
+    1.  Add 203.0.113.1/32 (default VRF) to NextHopGroup (default VRF) containing one NextHop (default VRF) specified to
```

Comment:
Since an implementation already exists for this test plan, please minimize the changes and follow the original structure as much as you can so the diff reflects only the intended material changes.

For example, at first it looked like you rewrote the whole test plan, but on closer reading you just swapped the lines for 198.51.100.0/24 and 203.0.113.1/32 and moved a bunch of lines around.

Reply:
Sorry, my bad for creating the confusion.

My intention is to make this test plan more clear. I should have split it into 2 PRs, 1) to add corrections/TODOs, 2) then reformat the wording and layout. 

Since you have already read and digest the whole changes, can you help LGTM this PR?